pkg/virtual/workspaces/registry/rest_test.go

```diff
@@ -361,64 +358,6 @@ func TestListWorkspacesWithUserPermission(t *testing.T) {
 	applyTest(t, test)
 }
 
-func TestListWorkspacesForbidden(t *testing.T) {
```

Comment:
Test are removed here, but I didn't see any test added to the new authorizer added in the `build.go` file. Did I miss it ?
Do you think there should be one ?

Reply:
added get/delete to e2e tests. List everybody with access has now.

Reply:
OK. Thanks !